### ISSUE TITLE
getConfigFromAnySource('unit.engine.tap.command') only pulls first command

### ISSUE THREAD

**Issue by elsigh** [tagview/arcanist-extensions#6]
If you have two tap tests this line won't work, it pulls the first matching config line that matches "unit.engine.tap.command".

Why not pull from actual runtime config for this class and just call the option "command" - seems cleaner, no?

**Comment by elsigh** [comment]
Oh I see, each engine doesn't have its own config. Maybe it should. ;0

**Comment by reu** [comment]
Yeah, that's a difficult problem, and it used to work (probably Arcanist changed it behaviour). Perhaps we should add something like a engine "name" when using MultiTestEngine, and then we could scope each config by this "name".

**Comment by elsigh** [comment]
It seems like the instance object of an engine should be able to hold a localized configuration somehow.
